string skill_path
string dialog
---
string response
---
# No feedback
